src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -147,15 +147,28 @@ export const getAllThorchainSaversPositions = async (
 export const getThorchainTransactionStatus = async (txHash: string) => {
   const thorTxHash = txHash.replace(/^0x/, '')
   const { data: thorTxData, status } = await axios.get<ThornodeStatusResponse>(
-    `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/tx/${thorTxHash}`,
+    `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/tx/status/${thorTxHash}`,
     // We don't want to throw on 404s, we're parsing these ourselves
     { validateStatus: () => true },
   )
 
   if ('error' in thorTxData || status === 404) return TxStatus.Unknown
-  if (!thorTxData.observed_tx.status || thorTxData.observed_tx.status === 'incomplete')
+  // Tx has been observed, but swap/outbound Tx hasn't been completed yet
+  if (
+    // Despite the Tx being observed, things may be slow to be picked on the THOR node side of things i.e for swaps to/from BTC
+    !(thorTxData.stages.inbound_observed?.completed === true) ||
```

Comment:
i suppose with this being the only required property, maybe we should keep it (even though it feels ilke `inbound_finalised` should always be returned as well...)
```suggestion
    thorTxData.stages.inbound_observed.completed === false ||
```